sfcasts/cor-sf-config-and-nullobject.md

```diff
@@ -8,9 +8,13 @@ We'll start by opening `CasinoHandler` and, above the class name, add
 the `#[Autoconfigure()]` attribute. When this class is *instantiated*, we want
 to call `setNext()` and pass another handler object. To do that, we'll use
 the `calls` option, so inside
-write `calls: [['setNext' => ['@'.LevelHandler::class]]]`. *So* when Symfony
-instantiates this class, it will call `setNext()` and pass a `LevelHandler` object.
-That's *it*!
+write `calls: [['setNext' => ['@'.LevelHandler::class]]]`. Note and be careful with this nested array syntax.
```

Comment:
This array syntax is a bit gnarly. Should I explain this a bit more of do you think my caution note is enough?

Reply:
yea I agree, this syntax is not easy to speak out. I like your note but I think it could be a bit shorter. What about `Be careful with this nested array syntax`?